<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arco</class>
 <widget class="QDialog" name="Arco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Arc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Radio (Y)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Radio (X)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Center (Y)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Center (X)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="radioX">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="radioY">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Y_Scale_Factor">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Reflection">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Scale Factor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Scale Factor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Reflected</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="X_Scale_Factor">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Scaled</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Reflection_Type">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vertical</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Translation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotating angle (degrees)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Translated</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Y_Rotating_Point">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>210</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="X_Translation">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Rotating Point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Rotated</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Rotating Point</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="X_Rotating_Point">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Translation</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Y_Translation">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Rotating_Angle">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>380</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw Normal</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="centerX">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="centerY">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Angle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>End Angle</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="startAngle">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="endAngle">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
